scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -785,6 +795,9 @@ private void processAlertRecord(final Record record, final Connection connection
 					errorHandler.onError(alert.getPeer(), alert.getLevel(), alert.getDescription());
```

Comment:
Not completely linked to this issue, but now we have MessageCallback.onError()`  + `SessionListener.handshakeFailed()`. Do we still need errorHandler ?